### PR TITLE
Remove .jform-rightbtn class

### PR DESCRIPTION
**Pull request by Quy**
### Summary of Changes
From #15092, per @C-Lodder, this class is not needed.

### Testing Instructions
Code review.

### CODE REVIEW

**Comment by RonakParmar** [comment]
I have tested this item :white_check_mark: successfully on effa15cc36647bf2362c6a1558016dcbea686607<br /><br />Done code review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15097">issues.joomla.org/tracker/joomla-cms/15097</a>.</sub>

**Comment by infograf768** [comment]
Do NOT take off the class from
administrator/components/com_finder/views/filter/tmpl/edit.php
but combine them as I did in https://github.com/joomla/joomla-cms/pull/15092

Do not modify components/com_finder/helpers/html/filter.php and leave it alone.

These are used both by Isis AND Hathor!.

OK for the change in administrator/components/com_templates/views/style/tmpl/edit_assignment.php as this one is overriden by Hathot

File: administrator/components/com_finder/views/filter/tmpl/edit.php [hunk 1]
```diff
@@ -70,7 +70,7 @@
 					<div class="well">
 						<?php echo $this->form->renderField('map_count'); ?>
 					</div>
-					<button class="btn btn-default" type="button" class="jform-rightbtn" onclick="jQuery('.filter-node').each(function () { this.click(); });">
+					<button class="btn btn-default" type="button" onclick="jQuery('.filter-node').each(function () { this.click(); });">
```
Comment (C-Lodder) [inline administrator/components/com_finder/views/filter/tmpl/edit.php]: `btn-default` isn't a BS2 class

**Comment by C-Lodder** [comment]
Seeing as this is used for Hathor, which already has it's own float classes, `jform-rightbtn` can be replaced with `fltrt`.

**Comment by infograf768** [comment]
I suggest to wait for a decision concerning Hathor.

See https://issues.joomla.org/tracker/joomla-cms/15079<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15097">issues.joomla.org/tracker/joomla-cms/15097</a>.</sub>

File: administrator/components/com_finder/views/filter/tmpl/edit.php (hunk 1 above)
Comment (brianteeman) [inline administrator/components/com_finder/views/filter/tmpl/edit.php]: and we use it 8 times :(

**Comment by franz-wohlkoenig** [comment]
@infograf768 Decision is made (comment by @wilsonge): "There has been a decision made. Hathor will continue to ship with J3.x however we do not guarantee feature parity of all Joomla features (i.e. everything that works now should continue to work but we don't guarantee new features like fields and associations etc will continue to work fully - although we won't obviously reject any PR's if they come in)"<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15097">issues.joomla.org/tracker/joomla-cms/15097</a>.</sub>

**Comment by infograf768** [comment]
@franz-wohlkoenig 
yep, i know that. but we should not kill what works fine either.
i have no issue if hathor is corrected at the same time.

**Comment by franz-wohlkoenig** [comment]
@infograf768  so this PR should be closed?

**Comment by infograf768** [comment]
it alreadt has conflicts, so i think it should be updated, and include the hathor changes. these are not new features, but css matters.

**Comment by Quy** [comment]
I will open a new pr.

**Pull request closed without merging**